Source/JavaScriptCore/wasm/WasmEntryPlan.cpp

```diff
@@ -119,27 +119,30 @@ void EntryPlan::prepare()
         || !tryReserveCapacity(m_unlinkedWasmToWasmCalls, functions.size(), " unlinked WebAssembly to WebAssembly calls"))
         return;
 
+    m_wasmToWasmExitStubs.resize(m_moduleInformation->importFunctionTypeIndices.size());
```

Comment:
Doesn’t this need to be inside `#if ENABLE(JIT)`?

Reply:
Indeed. I fixed it. Looks like all of the EWS bots enable JIT.